frontend/src/metabase/routes-stable-id-aware.tsx

```diff
@@ -123,6 +112,41 @@ export const EntityIdRedirect = ({
   }
 };
 
+function handleResults({
+  currentUrl,
+  paramsWithValues,
+  entity_ids,
+}: {
+  currentUrl: string;
+  paramsWithValues: ParamWithValue[];
+  entity_ids: TranslateEntityIdResponse | undefined;
+}) {
```

Comment:
Looks good! I feel like this function could have a small unit test, but not strictly needed.

Reply:
I initially thought of doing jest tests for this feature, but I realized that most issues were likely to happen because of things happening oh higher level (routing etc) so I ended up only doing e2e